src/main/java/chess/domain/board/RunningBoard.java

```diff
@@ -0,0 +1,84 @@
+package chess.domain.board;
+
+import chess.domain.piece.Blank;
+import chess.domain.piece.InitializedPawn;
+import chess.domain.piece.Piece;
+import chess.domain.piece.factory.PieceFactory;
+import chess.domain.piece.position.Position;
+import chess.domain.piece.team.Team;
+import chess.domain.ui.UserInterface;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RunningBoard extends StartedBoard {
+    private static final int LINE_START_INDEX = 1;
+    private static final int LINE_END_INDEX = 8;
+    private static final int BLACK_PAWN_ROW = 7;
+    private static final int WHITE_PAWN_ROW = 2;
+    private static final int BLANK_START_INDEX = 3;
+    private static final int BLANK_END_INDEX = 6;
+
+    RunningBoard(Map<Position, Piece> pieces, UserInterface userInterface) {
```

Comment:
질문 있습니다!
상속을 하려니 private 생성자를 쓸 수 없었습니다. 그래서 default 생성자로 만들었는데요.
아예 정적 팩토리 메소드를 없애고 생성자만 둘까도 생각해봤지만, 생성 시 초기화하는 것이라는 의미를 주고 싶어 정적팩토리 메소드를 사용하며 생성자를 default로 두었습니다.
정적팩토리 메서드 사용 시 생성자는 private으로 두는 게 원칙이라고 알고 있어, 이 코드에 문제가 있는 것 같은데요. 조금 더 나은 방식을 가르쳐주시면 감사하겠습다!

Reply:
```suggestion
    public static RunningBoard of(Map<Position, Piece> pieces, UserInterface userInterface) {
        return new RunningBoard(pieces, userInterface);
    }
```
RunningBoard에 팩토리 메서드를 추가하면 됩니다.

Reply:
Board를 상속 시켜야할 필요가 있는지 잘 모르겠네요
이펙티브 자바 3판 참고하세요
4장 클래스와 인터페이스
아이템 18. 상속보다는 컴포지션을 사용하라
아이템 19. 상속을 고려해 설계하고 문서화하라. 그러지 않았다면 상속을 금지하라
아이템 20. 추상 클래스보다는 인터페이스를 우선하라

Reply:
추상 부모 클래스에서 자식 클래스를 직접적으로 참조하면 강한 의존성이 생겨서 별로 권하고 싶지 않은 설계입니다.
다른 자식 클래스에도 영향을 미칠 수 있어요

Reply:
Board 쪽의 상속을 모두 제거했습니다~!
참고하라고 주신 이펙티브 자바 3편 아이템들 모두 읽어봤지만 거기에서 설명하는 부분을 아직 다 이해하진 못했지만요..^^;
감사합니다!